lib/ui/painting.dart

```diff
@@ -927,6 +932,8 @@ abstract class Codec extends NativeFieldWrapperClass2 {
 /// [list] is the binary image data (e.g a PNG or GIF binary data).
 /// The data can be for either static or animated images.
 ///
+/// The following image formats are supported: {@macro flutter.painting.imageFormats}
```

Comment:
The macro name doesn't match the template name?  The general strategy seems like it will work.  dartdoc currently has no limitations on what characters are allowed.

Reply:
oops.

Reply:
fixed.